### ISSUE TITLE
AVStream error & another

### ISSUE THREAD

**Issue by ChaeYoungKim-snu** [denisecailab/minian#66]
![AVStream_error](https://user-images.githubusercontent.com/47932984/95084291-f8876180-071d-11eb-8888-7ffdfe77d2e3.png) 

Hi,

Recently, from about 1month ago, I start to have a problem making videos(concatenated) with some error message. It seemed like it is ffmpeg related problem, I tried to fix it by downloading ffmpeg again but it did not work, so I want to report it to minian developers. I am using vlc player in Linux ubuntu.  

Here I post my error screenshot, with the final product minian.avi file with has white background only (unlike before which used to have 4 divided screens).

And another question: is there any way to change the code in my local computer so that I can generate my final videos after manual curation (delete or merge)? Now it is 'generate video'- 'curation'- then 'save new unit IDs', but these deleted or merged units are not fixed in the video or output display which has been already generated. Could you let me know which part I should change to get them after curation? (Or to plot cells that I am interested in only.) 

Thanks!

**Comment by phildong** [comment]
1. When generating the video everything is normalized to the max value of `Y`, so my guess for the white screen is that somehow your `Y` value is much smaller than expected. I'll suggest first looking at the numbers of `A` and `C` and `Y` to see if there's anything that seems weird. You can also manually pass a `scale` parameter to `generate_videos`, where 1 would correspond to no scaling. The "AVStream.codec" thing is only a warning and should be gone with an update of `pyav`

2. The source of `generate_videos` is under `visualization.py` so you can take a look at that and make it output whatever you want. However if all you want is to consider only the accepted cell, just pass in `A` and `C` with only those cells, for example, you can pass in something like `A.where(A.coords['unit_labels'] > 0, drop=True)` to `generate_videos` after manually rejecting cells.

**Comment by ChaeYoungKim-snu** [comment]
Hi!

1. I couldn't figure out the 1st problem as you said, but I just gave it a try with different video players and changed from VLC to SMPlayer, and it works again :) So I guess something with VLC player has changed and doesn't work properly in my PC.

2. I did as you commented, it was very useful! One thing I had to do extra was to generate minian dataset again to add 'unit_labels' after manually curating labels. And then I called the modified generate_videos (which I saved as generate_selected_videos). In the end, I got 2 videos, one with all cells and another with selected cells only.

Thank you for your help, I greatly appreciate it!
